docs/introduction.md

```diff
@@ -85,3 +85,72 @@ Available download clients and actions
 The development of autobrr started in Early 2020, entering rapid development in Summer 2021 due to dissatisfaction with needing 3+ tools to do one job. Autobrr has since gained quite a bit of traction and has a growing community supporting the project.
 
 Autobrr was developed with resource consumption in mind. The software uses API calls to reduce unnecessary downloads of .torrent files from sites like BTN, RED, PTP, and GGn. On other sites, it will download the .torrent only if the information is not present in the announce message.
+
+## Quick Start
+
+The following considers a common use case, feeding IRC announcements for a private
+tracker to a Servarr instance, as a way to illustrate how the components of autobrr fit
+together and what is required to get up and running.  It's not the only use case and
+other use cases may require more RTFM'ing in [the configuration docs](./configuration).
+
+1. [Install autobrr](./installation/linux).
+
+	Proceed once the web UI is accessible.
+
+2. Create a user for yourself.
+
+	Most easily done through the initial "wizzard" in the web UI when autobrr is first
+	started.
+
+3. [Register a nick on your indexer's IRC
+   network](https://autobrr.com/configuration/irc#registering-with-nickserv).
+
+4. [Group a "bot" nick with your real nick](./configuration/irc#grouping-nicks)
+
+5. Add an [indexer](./configuration/indexers).
+
+6. Add a [Servarr instance](./configuration/download-clients#sonarr)
+
+	**NOTE**: In the context of autobrr, Servarr instances are considered download
+	clients.
+
+7. Add a [filter](./filters).
+
+	To feed all IRC announcements to a Servarr instance to let it decide what, if
+	anything, to do with the release, just add a filter with the indexer from #5
+	selected and leave the rest blank.  Don't forget to enable the filter.
+
+	**NOTE**: Autobrr does nothing with received IRC announcements without at least one
+	filter applied to at least one indexer.
+
+8. Add a [filter action](./filters/actions).
+
+	Add an action of the matching Servarr type (e.g. Sonarr type action for a Sonarr
```

Comment:
Add an action of the matching download client.

Reply:
Ditto